src/cluster.jl

```diff
@@ -1,119 +1,93 @@
-function getGaussMembership(fixations, centers, sigma)
-    sqdist = pairwise(SqEuclidean(), fixations', centers')
-    gausskernels = exp.(-sigma^(-2.) * sqdist)
-    rownormalize(gausskernels)
-end
-
-rownormalize(M) = M ./ sum(M,2)
-
-# TODO: note this never really worked
-# create rate matrix
-function makeW(data, sigma, grid)
-    data[:index] = 1:size(data,1)
-    fixations = Array(data[[:x,:y]])
-    centers   = grid == :none ? fixations : grid
-    gaussmemb = getGaussMembership(fixations, centers, sigma)
-    n = size(centers,1)
-    ratesum = zeros(Float64, n, n)
-    fromsum = zeros(Float64, n)
-
-    by(data, :groupby) do subjdata
-        # user periodic boundary
-        from = gaussmemb[subjdata[1:size(subjdata,1)|>collect, :index],:]
-        to   = gaussmemb[subjdata[vcat(2:size(subjdata,1),1), :index],:]
-        time = 1./       subjdata[1:size(subjdata,1)|>collect, :time]
-        ratesum += from' * (to .* time)
-        fromsum += sum(from', 2)
-    end
-
-    W = ratesum ./ fromsum
-
-    for i=1:n
-        W[i,i] = -(sum(W[i,:]) - W[i,i])
-    end
-    return W
-end
-
-global SymmetrizeP = false
-setSymmetrizeP(b::Bool) = (SymmetrizeP = b)
-
-function makeP(data, tau, sigma, grid, symmetrize)
-    data[:index] = 1:size(data,1)
-    from = (Int64)[]
-    to   = (Int64)[]
-
-    by(data, :groupby) do subjdata
-        abstime = cumsum(subjdata[:time])
-        offs = subjdata[1,:index] - 1
-        time = 0
-        iold = 1
-        while (true)
-            time += tau
-            inew = findfirst(t-> (t>=time), abstime) # find nearest fixation after current

```

Comment:
This was wrong...
imagine starting at some point and staying there for a long time, inew still finds the next point over and over and thus counts a lot of selftransitions for the wrong point...